packages/material-ui/src/Avatar/Avatar.d.ts

```diff
@@ -55,7 +55,7 @@ declare const Avatar: OverridableComponent<AvatarTypeMap>;
 export type AvatarClassKey =
   | 'root'
   | 'colorDefault'
-  | 'circle'
+  | 'circular'
```

Comment:
Shouldn’t we just add the new class key instead of replacing the existing one to avoid breaking changes?

Reply:
We should. This was just a leftover.